BaseFilter/sections/antiadblock.txt

```diff
@@ -829,6 +829,10 @@ starwarsaddicted.it,wuyong.fun,26g.me,myqqjd.com,poapan.xyz,javboys.com,idevicec
 ##a[id][href="https://toolkitspro.com"][rel="noopener noreferrer"]
 ##a[id][href="https://chpadblock.com/"][rel="noopener noreferrer"]
 ! specific
+||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$script,xmlhttprequest,redirect=noopjs,domain=bombuj.si

```

Comment:
It is not CHP script.
Instead of redirect `#%#//scriptlet('prevent-fetch', 'pagead2.googlesyndication.com')` works here.
And please move to CZ section of foreign.txt